modules/ossm-creating-certificates-for-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,197 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-certificates-for-a-multi-cluster-mesh_{context}"]
+= Creating certificates for a multi-cluster mesh 
+
+This procedure explains how to create root and intermediate certificates for two clusters, named East and West.
```

Comment:
```suggestion
This procedure explains how to create the root and intermediate CA certificates for two clusters, named East and West.
```

Reply:
Also, "East" and "West" aren't the names of the two clusters, but simply a way to distinguish between two clusters in an east-west configuration.

Reply:
done